tests/core/basic20/task.yaml

```diff
@@ -58,4 +61,9 @@ execute: |
     done
 
     # ensure the "snap recovery" command works
-    snap recovery | MATCH '[0-9]+\ +canonical\*\ +ubuntu-core-20-[^ ]+\ +current'
+    MODEL="$(snap model --verbose | grep '^model' | awk '{ print $2 }')"
+    BRAND="$(snap model --verbose | grep '^brand' | cut -d' ' -f2- | xargs)"
+    if [ "$BRAND" = "canonical" ]; then
+        BRAND="canonical\*"
```

Comment:
I'm slightly confused by what are we trying to match in this case, can you give an example?

Reply:
@stolowski, thanks for review:
This is what we need to match:
google:ubuntu-core-20-64 .../tests/core/basic20# snap recovery --unicode never
Label Brand Model Notes
20200915 canonical* ubuntu-core-20-amd64-dangerous current

Reply:
Aah, so it's not a regex, we are literally matching "*", thanks.